src/Settings.php

```diff
@@ -227,6 +235,27 @@ public static function getPathsFromStdIn()
         $lines = explode("\n", rtrim($content));
         return array_map('rtrim', $lines);
     }
+
+    public function __get($property)
```

Comment:
It is not necessary to be backward compatible here for `json`property

Reply:
No problem!
Would exposing `format` and adjusting tests that rely on `json` accordingly be OK? I aimed for not changing existing tests for `Settings` (assuming that it's kind of a contract that shouldn't be broken) but will be glad to go either way.